README.md

```diff
@@ -47,3 +47,11 @@ JShellAPI is a REST API, and whenever some code is received, it will create a se
 
 ## How to use JShellApi ?
 See [JShellAPI README](JShellAPI/README.MD)
+
+## Testing
+**JShellWrapper** Docker image lifecycle is handled during tests.
+
+The image name is injected from the root [build.gradle](./build.gradle) file, to JShellAPI project's [build.gradle](./JShellAPI/build.gradle) file and also to its [application.yaml](./JShellAPI/src/main/resources/application.yaml).
+
+- JShellWrapper Docker Image is built before the tests run.
+- **Container & Cleanup**: After the tests finish, the container and image are removed to ensure a clean environment.
```

Comment:
How many times I have to repeat this, readme isn't documentation nor user manual, you are supposed to tell the user how to run the tests, not how the project works.

Reply:
oh *** i forgot it